tests/sampling/test_jax.py

```diff
@@ -229,7 +229,7 @@ def test_get_log_likelihood():
     b_true = trace.log_likelihood.b.values
     a = np.array(trace.posterior.a)
     sigma_log_ = np.log(np.array(trace.posterior.sigma))
-    b_jax = _get_log_likelihood(model, [a, sigma_log_])["b"]
+    b_jax = jax.vmap(_get_log_likelihood_fn(model))([a, sigma_log_])["b"]
```

Comment:
why did the behavior (had to) change?

Reply:
For postprocessing I needed to be able to calculate the log_likelihood without the final wrapping vmap.  It's possible to have it just calculate the likelihood instead of returning a function.  However the extra vmap will still be necessary

Reply:
Changed to calculate the likelihood instead of using a returned likelihood calculator function.